packages/rollup-full-node/src/app/utils/utils.ts

```diff
@@ -121,3 +121,12 @@ export async function deployContract(
   await contract.deployed()
   return contract
 }
+
+/**
+ * Gets the current number of seconds since the epoch.
+ *
+ * @returns The seconds since epoch.
+ */
+export function getTimestamp(): number {
```

Comment:
Let's rename this function to getCurrentTime(), since this is not actually getting timestamp